### PR TITLE
API for workshop survey results now includes free response answers

### PR DESCRIPTION
**Pull request by mehalshah**
image

### CODE REVIEW

File: dashboard/lib/controllers/api/v1/pd/workshop_score_summarizer.rb [hunk 1]
```diff
@@ -107,10 +127,14 @@ def get_score_for_workshops(workshops, facilitator_breakdown = false)
     report_rows[:teacher_engagement] = (report_rows[:teacher_engagement] / (TEACHER_ENGAGEMENT_QUESTIONS.size.to_f * response_count)).round(2)
     report_rows[:overall_success] = (report_rows[:overall_success] / (OVERALL_SUCCESS_QUESTIONS.size.to_f * response_count)).round(2)
 
-    INDIVIDUAL_RESPONSE_QUESTIONS.each do |question|
+    INDIVIDUAL_RESPONSE_QUESTIONS.map.each do |question|
```
Comment (aoby) [inline dashboard/lib/controllers/api/v1/pd/workshop_score_summarizer.rb]: Why map?

File: dashboard/lib/controllers/api/v1/pd/workshop_score_summarizer.rb [hunk 2]
```diff
@@ -81,6 +96,7 @@ def get_score_for_workshops(workshops, facilitator_breakdown = false)
         survey_response.symbolize_keys.each do |k, v|
           if OVERALL_SUCCESS_QUESTIONS.include?(k)
             score = PdWorkshopSurvey::AGREE_SCALE_OPTIONS.index(v) + 1
+          elsif FREE_RESPONSE_QUESTIONS.include?(k)
           else
```
Comment (aoby) [inline dashboard/lib/controllers/api/v1/pd/workshop_score_summarizer.rb:101]: Is this if clause intentionally empty? (betw lines 99-100 on the new file)
If so, why not change the else to `elsif !FREE_RESPONSE_QUESTIONS.include?(k)` instead? Or at least add a comment here explaining that these are being intentionally skipped.

Comment (mehalshah) [reply dashboard/lib/controllers/api/v1/pd/workshop_score_summarizer.rb:101]: It's intentionally empty, I don't want to compute score or do anything. Happy to comment

File: dashboard/lib/controllers/api/v1/pd/workshop_score_summarizer.rb (hunk 1 above)
Comment (mehalshah) [reply dashboard/lib/controllers/api/v1/pd/workshop_score_summarizer.rb]: Leftover from earlier, will remove

**Pull request merged**